api/notd/model.py

```diff
@@ -175,3 +175,40 @@ class TokenMultiOwnership(RetrievedTokenMultiOwnership):
     tokenMultiOwnershipId: int
     createdDate: datetime.datetime
     updatedDate: datetime.datetime
+
+@dataclasses.dataclass
+class RetrievedCollectionHourlyActivity:
+    address: str
+    date: datetime.datetime
```

Comment:
this is wierd - it should have start and end, not just date

Reply:
ah right yes sorry

Reply:
And i should make this a date range?, which means or message will also have start and end date?

Reply:
Ah acutally yes it makes sense just as Date cos it has Hour in the class name